### ISSUE TITLE
testem config causes hang on headless chrome >= 76

### ISSUE THREAD

**Issue by oliverlangan** [ember-cli/ember-cli#8778]
(This was discussed in the #topic-testing discord channel.)

The `testem.js` file included in the Ember CLI blueprints includes two chrome flags which cause headless Chrome >= 76 to hang with the error:
```
Built project successfully. Stored in "/Users/langan/git/hwet/balance_ui/tmp/class-tests_dist-EsVo3LRj.tmp".
not ok 1 Chrome - [undefined ms] - error
    ---
        message: >
            Error: Browser failed to connect within 30s. testem.js not loaded?
            Stderr: 
             
            DevTools listening on ws://0.0.0.0:9222/devtools/browser/b88b8d02-b089-4ac1-83d0-c1a9edcb831f
            [0802/175453.402703:WARNING:process_memory_mac.cc(93)] mach_vm_read(0x7ffee75ca000, 0x2000): (os/kern) invalid address (1)
            [0802/175453.725255:WARNING:system_snapshot_mac.cc(42)] sysctlbyname kern.nx: No such file or directory (2)
            [0802/175453.772798:WARNING:gpu_process_host.cc(1188)] The GPU process has crashed 1 time(s)
...
```

Removing _either_ of these two flags fixes the issue:
```
          '--disable-gpu',
          '--disable-software-rasterizer',
``

Per the Discord conversation, the `--disable-gpu` option was added to address [an issue on Windows](https://bugs.chromium.org/p/chromium/issues/detail?id=737678), which has since been solved. That flag can probably be safely removed.

**Comment by kellyselden** [comment]
Resolved in #8774.

**Comment by oliverlangan** [comment]
d'oh! I didn't see an issue for it, should have looked at the PRs. Thanks @kellyselden!
